packages/sanes-browser-extension/config/manifest-production.json

```diff
@@ -20,7 +20,12 @@
   },
   "content_scripts": [],
   "externally_connectable": {
-    "matches": ["http://localhost/*", "https://*.iov.one/*", "https://*.neuma.io/*"]
+    "matches": [
+      "http://localhost/*",
+      "https://*.iov.one/*",
+      "https://*.neuma.io/*",
+      "https://*.starname.network/*"
```

Comment:
Can you verify if this also matches https://starname.network (i.e. no subsomain)? This is probably documented in the Google extension docs.

We don't need non-subdomains for the other domains but we might want to use it here. I'm surprised we redirect to www.starname.network at the moment. We better prepare for using pure starname.network as well.

Reply:
It should work?
![image](https://user-images.githubusercontent.com/44572727/70707203-e5f7f380-1cd7-11ea-851a-5465d8ff864b.png)

Reply:
Looks good, thanks for checking. If it does not work it is a Google bug ;)